statsmodels/tsa/tests/test_arima.py

```diff
@@ -759,191 +835,197 @@ def test_arma_predict_mle_dates():
-            method='css', trend='nc')
+    res1 = ARIMA(cpi, (4, 1, 1), dates=cpi_dates, freq='Q').fit(disp=-1,
+                                                                method='css',
+                                                                trend='nc')
 
-    params = np.array([ 1.231272508473910,
+    params = np.array([1.231272508473910,
                        -0.282516097759915,
-                        0.170052755782440,
+                       0.170052755782440,
                        -0.118203728504945,
                        -0.938783134717947])
 
-    with open(current_path + '/results/results_arima_forecasts_all_css.csv', "rb") as test_data:
-        arima_forecasts = np.genfromtxt(test_data, delimiter=",", skip_header=1, dtype=float)
+    with open(current_path + '/results/results_arima_forecasts_all_css.csv',
+              "rb") as test_data:
+        arima_forecasts = np.genfromtxt(test_data, delimiter=",",
+                                        skip_header=1, dtype=float)
 
-    fc = arima_forecasts[:,0]
-    fcdyn = arima_forecasts[:,1]
-    fcdyn2 = arima_forecasts[:,2]
+    fc = arima_forecasts[:, 0]
+    fcdyn = arima_forecasts[:, 1]
+    fcdyn2 = arima_forecasts[:, 2]
 
     start, end = 5, 51
     fv = res1.model.predict(params, '1960Q2', '1971Q4', typ='levels')
-    assert_almost_equal(fv, fc[start:end+1], DECIMAL_4)
-    assert_equal(res1.data.predict_dates, cpi_dates[start:end+1])
+    assert_almost_equal(fv, fc[start:end + 1], DECIMAL_4)
+    assert_equal(res1.data.predict_dates, cpi_dates[start:end + 1])
 
     start, end = 202, 227
     fv = res1.model.predict(params, '2009Q3', '2015Q4', typ='levels')
-    assert_almost_equal(fv, fc[start:end+1], DECIMAL_4)
+    assert_almost_equal(fv, fc[start:end + 1], DECIMAL_4)
     assert_equal(res1.data.predict_dates, cpi_predict_dates)
 
     # make sure dynamic works
     start, end = 5, 51
     fv = res1.model.predict(params, '1960Q2', '1971Q4', typ='levels',
-                                                        dynamic=True)
-    assert_almost_equal(fv, fcdyn[start:end+1], DECIMAL_4)
+                            dynamic=True)
+    assert_almost_equal(fv, fcdyn[start:end + 1], DECIMAL_4)
 
     start, end = '1965q1', '2015q4'
     fv = res1.model.predict(params, start, end, dynamic=True, typ='levels')
-    assert_almost_equal(fv, fcdyn2[24:227+1], DECIMAL_4)
+    assert_almost_equal(fv, fcdyn2[24:227 + 1], DECIMAL_4)
 
 
 def test_arma_predict_css_dates():
     from statsmodels.datasets.sunspots import load_pandas
     sunspots = load_pandas().data['SUNACTIVITY'].values
-    mod = ARMA(sunspots, (9,0), dates=sun_dates, freq='A')
+    mod = ARMA(sunspots, (9, 0), dates=sun_dates, freq='A')
     mod.method = 'css'
-    assert_raises(ValueError, mod._get_prediction_index, *('1701', '1751', False))
+    assert_raises(ValueError, mod._get_prediction_index,
+                  *('1701', '1751', False))
```

Comment:
pytest.raises; the *arg expansion here is ugly

Reply:
Not going to switch to pytest, but agree about *